src/components/organisms/tables/AddUsersTable/AddUsersTable.tsx

```diff
@@ -87,7 +87,6 @@ const AddUsersTable: FC<AddUsersTableProps> = ({
               control={control}
               rowErrors={rowsWithErrors[`row-${row?.index}`]}
               errorZIndex={errorZIndex}
-              type="number"
```

Comment:
**[non-blocker]**

This also removes the numbers constraint from the input, making it more prone to errors. Not a massive problem right now, but there are also ways of just hiding the arrows for type="number" inputs. https://www.w3schools.com/howto/howto_css_hide_arrow_number.asp - 1 example here

Reply:
Good point! Switched to css solution 👍